pagopa/pn-frontend
# Fix/pn 5274 fe aggiornare il compiler target delle webapp

## Short description
Fixed to ES2020 target compiler for all webapp (except landing webapp)

## How to test
Check that build and compiler tasks succeed. Check confluence page on motivation around this choice.
